.github/workflows/clean-pr-description-on-approval.yml

```diff
@@ -0,0 +1,51 @@
+name: Clean PR Description on First Approval
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  strip-comments:
+    if: github.event.review.state == 'approved'
```

Comment:
What makes this on first approval? That value is the overall approved status, not the type of review submitted?